Header header

float64 px
float64 py

Feature_Keypoint f

float64 posX
float64 posY
float64 posZ
